sdk/core/Azure.Core.Experimental/tests/LowLevelClientTests.cs

```diff
@@ -77,19 +77,63 @@ public void CannotGetOutputModelOnFailureCodeAsync()
             Assert.ThrowsAsync<RequestFailedException>(async () => await client.GetPetAsync("pet1"));
         }
 
-        #region Helpers
+		// Note: these two tests *currently* test different code paths:
+		// 1) In the ResponseStatusOptions.Default case, we're going through code paths in 
+		//    ClientDiagnostics that we'll later migrate to ResponseClassifier (see https://github.com/azure/azure-sdk-for-net/issues/24031)
+        //
+		//    Because this one is thrown from client's `GetPetAsync()` method, where it calls
+		//    _clientDiagnostics.CreateRFException() -- this happens via different constructors (Note: does it have to?  Could we refactor this?)
+		// 
+		// 2) In the ResponseStatusOptions.NoThrow case, we're going through code paths in
+		//    ResponseClassifier, which will become the only path after resolution of #24031
```

Comment:
Note: we'll reconcile these two code paths in https://github.com/azure/azure-sdk-for-net/issues/24031